medicines/api/src/product.rs

```diff
@@ -56,12 +56,8 @@ pub fn handle_doc(document: &IndexResult, products: &mut Vec<Product>) {
                 .iter_mut()
                 .find(|product| document_product_name == &product.name);
 
-            match existing_product {
```

Comment:
seems unnecessary to remove this if we're planning to add back a version of it – we could use the filters in the `documents()` call to filter a pre-existing `documents` list on the `Product`

Reply:
Yeah, you'd still need to make some other changes though so I think we might as well just put it back later